package.json

```diff
@@ -28,20 +28,20 @@
   },
   "homepage": "https://github.com/tensorflow/tensorboard#readme",
   "devDependencies": {
-    "@angular/bazel": "^9.0.0",
-    "@angular/cli": "^9.1.2",
-    "@angular/compiler": "^9.0.0",
-    "@angular/compiler-cli": "^9.0.0",
-    "@bazel/bazel": "2.1",
-    "@bazel/ibazel": "^0.12.4",
-    "@bazel/jasmine": "^1.6.0",
-    "@bazel/karma": "1.6.0",
-    "@bazel/protractor": "^1.6.0",
-    "@bazel/rollup": "^1.6.0",
-    "@bazel/terser": "^1.6.0",
-    "@bazel/typescript": "^1.6.0",
-    "@rollup/plugin-commonjs": "^14.0.0",
-    "@rollup/plugin-node-resolve": "^8.4.0",
+    "@angular/bazel": "10.2.3",
+    "@angular/cli": "^10.1.1",
+    "@angular/compiler": "^10.1.2",
+    "@angular/compiler-cli": "^10.1.2",
+    "@bazel/ibazel": "^0.14.0",
+    "@bazel/jasmine": "^2.3.0",
+    "@bazel/karma": "^2.3.0",
+    "@bazel/protractor": "^2.3.0",
+    "@bazel/rollup": "^2.3.0",
+    "@bazel/terser": "^2.3.0",
+    "@bazel/typescript": "^2.3.0",
+    "@rollup/plugin-commonjs": "^16.0.0",
+    "@rollup/plugin-node-resolve": "^10.0.0",
+    "@types/chai": "^4.2.14",
```

Comment:
Is `types/chai` still needed, now that our active tests were moved to jasmine after https://github.com/tensorflow/tensorboard/pull/4392 ?